source/write/run-command.txt

```diff
@@ -0,0 +1,180 @@
+statistics, initializing a replica set, or running an aggregation pipeline.
+
+.. important:: Prefer Library Methods to Database Commands
+
+   The library provides wrapper methods for many database commands.
+   We recommend using these methods instead of executing database
+   commands when possible.
+   
+   To perform administrative tasks, use the :mongosh:`MongoDB Shell </>`
+   instead of the {+php-library+}. Calling the ``db.runCommand()``
+   method inside the shell is the preferred way to issue database
+   commands, as it provides a consistent interface between the shell and
+   drivers or libraries.
+
+.. _php-execute-command:
+
+Execute a Command
+-----------------
+
+To run a database command, you must specify the command and any relevant
+parameters in a command document, then pass the command document to the
+``MongoDB\Database::command()`` method. This method returns a
+``Cursor`` object.
+
+The following code shows how you can use the ``command()``
+method on a ``MongoDB\Driver\Database`` instance to run the ``hello``
+command, which returns information about the current member's role in
+the replica set:
+
+.. code-block:: php
+
+   $database = $client->selectDatabase('<db>');
+   $result = $database->command(['hello' => 1]);
+
+To find a link to a full list of database commands and corresponding
+parameters, see the :ref:`Additional Information section
+<php-addtl-info-runcommand>`.
+
+.. note:: Read Preference
+
+   The ``command()`` method does not obey the read preference you might
+   have set on your ``Database`` instance elsewhere in your code. By
+   default, ``command()`` uses the ``primary`` read preference.
+   
+   You can set a read preference for command execution by setting one
+   in the options parameter, as shown in the following code:
+   
+   .. code-block:: php
+
+      $readPref = new MongoDB\Driver\ReadPreference('primaryPreferred');
+      $result = $database->command(
+          ['hello' => 1], 
+          ['readPreference' => $readPref]
+      );
+
+   For more information on read preference options, see :manual:`Read
+   Preference </core/read-preference/>` in the {+mdb-server+} manual.
+
+.. _php-command-response:
+
+Response
+--------
+
+The ``command()`` method returns a ``Cursor`` object that contains
+the response from the database after the command has been executed.
+
+Each database command performs a different function, so the response
+content can vary depending on the command executed. However, every
+response contains a document with the following fields:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Field
+     - Description
+
+   * - <command result>
+     - Fields specific to the database command. For example,
+       ``count`` returns the ``n`` field and ``explain`` returns the
+       ``queryPlanner`` field.
+
+   * - ``ok``
+     - Whether the command has succeeded (``1``)
+       or failed (``0``).
```

Comment:
You can note that the PHP driver will raise exceptions if the `ok` field is zero. It will likely be [CommandException](https://www.php.net/manual/en/class.mongodb-driver-exception-commandexception.php), which exposes a `getResultDocument()` method to obtain the command result document.

That behavior happens long before the PHP driver would try to unpack a command result with a cursor envelope, as a cursor envelope is part of a successful response and would already have an `ok: 1` field.